cmd/opm/alpha/veneer/render.go

```diff
@@ -0,0 +1,82 @@
+package veneer
+
+import (
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"github.com/operator-framework/operator-registry/alpha/declcfg"
+	"github.com/operator-framework/operator-registry/alpha/veneer/basic"
+	"github.com/operator-framework/operator-registry/cmd/opm/internal/util"
+	containerd "github.com/operator-framework/operator-registry/pkg/image/containerdregistry"
+)
+
+func newBasicVeneerRenderCmd() *cobra.Command {
+	var (
+		veneer basic.BasicVeneer
+		output string
+	)
+	cmd := &cobra.Command{
+		Use:   "render veneer-file...",
+		Short: "Generate a declarative config blob from one or more veneer files",
+		Long:  `Generate a declarative config blob from one or more veneer files`,
+		Args:  cobra.MinimumNArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+
+			var write func(declcfg.DeclarativeConfig, io.Writer) error
+			switch output {
+			case "yaml":
+				write = declcfg.WriteYAML
+			case "json":
+				write = declcfg.WriteJSON
+			default:
+				log.Fatalf("invalid --output value %q, expected (json|yaml)", output)
+			}
```

Comment:
You could mark this `output` flag as required and add another layer of validation to it.

For an example of making a flag required:
https://github.com/operator-framework/operator-registry/blob/9e59597a435341d857283011c8ede1c16e5e292b/cmd/opm/alpha/bundle/build.go#L49

Reply:
I'm following the pattern of the 'opm alpha render' command, where we default to JSON output unless countermanded with this option.